migrations/2020-11-05-112238_initialize_table/down.sql

```diff
@@ -0,0 +1,9 @@
+-- This file should undo anything in `up.sql`
+DROP TYPE IF EXISTS Laporan_status;
+
+DROP TABLE IF EXISTS public.attachment;
+DROP TABLE IF EXISTS public.laporan;
+DROP TABLE IF EXISTS public.pelapor;
+DROP TABLE IF EXISTS public.referensi;
+DROP TABLE IF EXISTS public.satuan_kerja;
+DROP TABLE IF EXISTS public.user;
```

Comment:
How to resolve error on `diesel migration redo`
```suggestion
DROP TABLE IF EXISTS public.attachment;
DROP TABLE IF EXISTS public.laporan;
DROP TABLE IF EXISTS public.pelapor;
DROP TABLE IF EXISTS public.referensi;
DROP TABLE IF EXISTS public.satuan_kerja;
DROP TABLE IF EXISTS public.user;

DROP TYPE IF EXISTS Laporan_status;

```

Reply:
saya terima sugestion changenya. @BimaAdi :sunglasses: